controls/postgres_spec.rb

```diff
@@ -156,29 +103,29 @@
   end
 end
 
-control 'postgres-06' do
+control 'postgres-07' do
   impact 1.0
   title 'Use salted hash to store postgresql passwords'
   desc 'Store postgresql passwords in salted hash format (e.g. salted MD5).'
   case postgres.version
   when /^9/
     describe postgres_session(USER, PASSWORD).query('SELECT passwd FROM pg_shadow;') do
-      its('output') { should match(/^md5\S*$/) }
+      its('output') { should match(/^md5\S*$/i) }
```

Comment:
style issue. Discussed in other baseline, we want to avoid extra paranteses

```suggestion
      its('output') { should match /^md5\S*$/i }
```